src/renderer/components/menu/menu-actions.tsx

```diff
@@ -70,6 +71,10 @@ export class MenuActions extends React.Component<MenuActionsProps> {
       ...(typeof triggerIcon === "object" ? triggerIcon : {}),
     };
 
+    if (this.props.onOpen) {
+      iconProps.onClick = this.props.onOpen;
+    }
+
```

Comment:
If this is undefined it should still be fine and `onOpen` can be destructured as on line 58.

```suggestion
      onClick: onOpen,
      ...(typeof triggerIcon === "object" ? triggerIcon : {}),
    };
```